strapi/strapi
# Resizing fields in edit view

![strapi_resize](https://user-images.githubusercontent.com/1172672/113051140-f0c4e900-91a5-11eb-967e-21d58167c3fb.gif)


### What does it do?

- Cleanup content-manager (DraggedField to flex without absolute positioning, duplicated css, Tooltip.js not used/working anymore)
- Added resize function to edit view fields except dynamiczones and components (minSize: 2, maxSize: 12)

### Why is it needed?

Using 50% of the remaining space for a single dropdown or radio button makes editing complicated because much space is wasted.

### How to test it?

Edit view of any component. A new button is shown at the end of the fields. By dragging the element the field could be resized to any size between 2/12 and 12/12. 2/12 instead of 1/12 because 1/12 is to small for description and buttons. Maybe someone has an idea how to handle too small elements.

### Related issue(s)/PR(s)

Issues: #8094, #8950
Strapi Community Discussion: 
https://forum.strapi.io/t/change-width-of-fields-on-edit-page/181
https://forum.strapi.io/t/columns-in-content-manager-detail-view/1011/2
